Sources/GRPC/ConnectionManager.swift

```diff
@@ -860,6 +891,62 @@ extension ConnectionManager {
   }
 }
 
+extension ConnectionManager {
+  /// Returns a synchronous view of the connection manager; each operation requires the caller to be
+  /// executing on the same `EventLoop` as the connection manager.
+  internal var sync: Sync {
+    return Sync(self)
+  }
+
+  internal struct Sync {
```

Comment:
Nit: I know this isn't public, but I'm not in love with the name `Sync`

Reply:
"sync" is _sort of_ common enough now (synchronous channel options, synchronous pipeline operations) to have meaning (i.e. you must be on the right event loop). That said, if you have a better name I'm all ears!